code/obj/machinery/manufacturer.dm

```diff
@@ -32,18 +32,20 @@
 	// 0 is =>, 1 is ==
 	var/base_material_class = /obj/item/material_piece/ // please only material pieces
 	var/obj/item/reagent_containers/glass/beaker = null
+	var/obj/item/fabdisk = null
 	var/list/resource_amounts = list()
 	var/area_name = null
 	var/output_target = null
 	var/list/materials_in_use = list()
 	var/list/available = list()
 	var/list/download = list()
+	var/list/disk = list()
```

Comment:
rename to `from_disk` or something for clarity please

Reply:
Renamed to disk_recipes